apps/next/components/Links.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+import Github from "../public/github.png";
+import Discord from "../public/discord.png";
+import Twitter from "../public/twitter.png";
+import Website from "../public/website.png";
+import Docs from "../public/docs.png";
+
+function Links() {
+  return (
+    <>
+      <Link
+        target="_blank"
+        href="https://github.com/flaredomains"
+        className="shrink-0"
+      >
+        <div
+          className={`hover:scale-125 transform transition duration-400 ease-out`}
+        >
+          <Image
+            width={32}
+            height={32}
+            className="h-4 w-auto opacity-80 hover:opacity-100"
+            src={Github}
+            alt="FNS"
+          />
+        </div>
+      </Link>
+      <Link
+        target="_blank"
+        href="https://discord.gg/wDd3pGsscZ"
+        className="shrink-0"
+      >
+        <div
+          className={`hover:scale-125 transform transition duration-400 ease-out`}
+        >
+          <Image
+            width={32}
+            height={32}
+            className="h-4 w-auto opacity-80 hover:opacity-100"
+            src={Discord}
+            alt="FNS"
+          />
+        </div>
+      </Link>
+      <Link
+        target="_blank"
+        href="https://twitter.com/flarenaming"
+        className="shrink-0"
+      >
+        <div
+          className={`hover:scale-125 transform transition duration-400 ease-out`}
+        >
+          <Image
+            width={32}
+            height={32}
+            className="h-4 w-auto opacity-80 hover:opacity-100"
+            src={Twitter}
+            alt="FNS"
```

Comment:
If we're going to use alternative text for these images, the text should actually describe what the image is supposed to be. Please change the alt text for each to be specific for GitHub, Discord, Twitter, etc. not just "FNS" for all of them.